src/Microsoft.AspNetCore.SignalR.Common/Protocol/IHubProtocol.cs

```diff
@@ -18,10 +18,15 @@ public interface IHubProtocol
         string Name { get; }
 
         /// <summary>
-        /// Gets the version of the protocol.
+        /// Gets the major version of the protocol.
         /// </summary>
         int Version { get; }
 
+        /// <summary>
+        /// Gets the minor version of the protocol.
+        /// </summary>
+        int MinorVersion { get; }
```

Comment:
@muratg for breaking change approval in 3.0. This is a new property that *hub protocols* will need to implement.

Reply:
:shipit: (in 3.0) :)